framework/src/components/storage/sql/migrations/updates/20190215094703_remove_uncofirmed_state.sql

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2018 Lisk Foundation
```

Comment:
probably filename for this needs to be updated later according to 1.6

Reply:
You mean due to the new migrations added recently right?

Reply:
yes!

Reply:
We will need to deal with this once we start merges of ongoing PRs I think. What you think?

Reply:
Resolving this one. I will create an issue for resolving migration naming that should be worked on before 1.7 release.